Src/Screens/OfferScreen/Viewables/OfferStartDateButton.swift

```diff
@@ -0,0 +1,134 @@
+        let bag = DisposeBag()
+        
+        let containerStackView = UIStackView()
+        containerStackView.alignment = .center
+        containerStackView.axis = .vertical
+        
+        let button = UIControl()
+        button.layer.borderWidth = 1
+        bag += button.applyBorderColor { traitCollection -> UIColor in
+            return .white
+        }
+        bag += button.applyCornerRadius { _ -> CGFloat in
+            return button.layer.frame.height / 2
+        }
+        
+        containerStackView.addArrangedSubview(button)
+        
+        bag += containerScrollView.contentOffsetSignal.onValue { contentOffset in
+            containerStackView.transform = CGAffineTransform(
+                translationX: 0,
+                y: (contentOffset.y / 5)
+            )
+        }
+        
+        let touchUpInside = button.signal(for: .touchUpInside)
+        bag += touchUpInside.feedback(type: .impactLight)
+        
+        let present = PresentStartDate()
+        
+        bag += touchUpInside.onValue({ _ in
+            bag += self.presentingViewController.present(
+                DraggableOverlay(presentable: present,
+                                 presentationOptions: [.defaults , .prefersNavigationBarHidden(true)]
+                )
+            ).disposable
+        })
+        
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.distribution = .equalSpacing
+        stackView.alignment = .center
+        stackView.spacing = 8
+        stackView.layoutMargins = UIEdgeInsets(top: 12, left: 20, bottom: 12, right: 20)
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.isUserInteractionEnabled = false
+        button.addSubview(stackView)
+        
+        stackView.snp.makeConstraints { make in
+            make.top.bottom.leading.trailing.equalToSuperview()
+        }
+        
+        let keyLabel = UILabel(value: "Startdatum", style: .body)
+        stackView.addArrangedSubview(keyLabel)
+        keyLabel.textColor = .white
+        
+        let valueLabel = UILabel(value: "", style: .bodyBold)
+            valueLabel.textColor = .white
+            stackView.addArrangedSubview(valueLabel)
+        
+            let iconView = Icon(icon: Asset.chevronRightWhite, iconWidth: 20)
+            iconView.transform = CGAffineTransform(rotationAngle: CGFloat(Double.pi/2))
+            stackView.addArrangedSubview(iconView)
+        
+        bag += self.client.watch(query: HasStartDateQuery()).onValue { result in
+            let startDate = result.data?.lastQuoteOfMember.asCompleteQuote?.startDate
+            
+            let today = Date()
+            let formatter = DateFormatter()
+            formatter.dateStyle = .short
+            formatter.dateFormat = "yyyy-MM-dd"
+            let todayString = formatter.string(from: today)
+        
+            if startDate == todayString {
```

Comment:
Use a `Calendar` to check this instead of formatting it, convert `startDate` to a `Date` instance with the formatter and then you could do https://developer.apple.com/documentation/foundation/calendar/2293243-isdateintoday